code/lib/cli/src/sandbox-templates.ts

```diff
@@ -432,6 +432,26 @@ const baseTemplates = {
     // TODO: The community template does not provide standard stories, which is required for e2e tests.
     skipTasks: ['e2e-tests', 'e2e-tests-dev', 'bench'],
   },
+  'ember/3-js': {
+    name: 'Create Ember 3 app (Javascript)',
+    script: 'npx --package ember-cli@3.28.1 ember new {{beforeDir}}',
+    inDevelopment: true,
+    expected: {
+      framework: '@storybook/ember',
+      renderer: '@storybook/ember',
+      builder: '@storybook/builder-webpack5',
+    },
+  },
+  'ember/default-js': {
+    name: 'Create Ember 4 app (Javascript)',
+    script: 'npx --package ember-cli@4.12.1 ember new {{beforeDir}}',
+    inDevelopment: true,
+    expected: {
+      framework: '@storybook/ember',
+      renderer: '@storybook/ember',
+      builder: '@storybook/builder-webpack5',
+    },
+  },
```

Comment:
We might need to change the CircleCI config because of the changed number of sandboxes.
And also create chromatic projects and add the project-tokens to the CI environment variables.

Reply:
Generating a working sandbox require manual intervention (to install ember-cli-storybook mainly). How can I do that?

Reply:
@yannbf Are these newly added templates not running yet due to the inDevelopment boolean?
I forgot how this works, can you help remind me.
At which point do we need to do the above? (upping the parallelism in CI and create the env-variables/projects to get chromatic on the templates working)

Reply:
They must be added in the exported `normal` const in `sandbox-templates.ts`: https://github.com/storybookjs/storybook/blob/e59d2c95ce8a96569e9bc2033e5e29f7e8c7fe2b/code/lib/cli/src/sandbox-templates.ts#L537

Reply:
> Generating a working sandbox require manual intervention (to install ember-cli-storybook mainly). How can I do that?

What sort of manual intervention is needed? 
Should it be scripted into the `init` command?
Should it be scripted in the `script` of the template?
https://github.com/storybookjs/storybook/blob/c1b4e2ef5d7662ec5abb08df316b45a4b0f416e0/code/lib/cli/src/sandbox-templates.ts#L99-L100

Reply:
`[ember-cli-storybook](https://github.com/storybookjs/ember-cli-storybook)` need to to installed and patched with https://github.com/storybookjs/ember-cli-storybook/pull/183.